cmd/apps.go

```diff
@@ -135,6 +138,40 @@ var lsKonnectorsCmd = &cobra.Command{
 	},
 }
 
+var runKonnectorsCmd = &cobra.Command{
+	Use:   "run [slug]",
+	Short: "Run a konnector.",
+	Long:  "Run a konnector named with specified slug using the specified options.",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if flagAppsDomain == "" {
+			errPrintfln("%s", errAppsMissingDomain)
+			return cmd.Usage()
+		}
+		if len(args) < 1 {
+			return cmd.Usage()
+		}
+		slug := args[0]
+		c := newClient(flagAppsDomain, consts.Jobs+":POST:konnector:worker")
+		j, err := c.JobPush(&client.JobOptions{
+			Worker: "konnector",
+			Arguments: map[string]interface{}{
+				"konnector":      slug,
+				"account":        flagKonnectorAccount,
+				"folder_to_save": flagKonnectorFolder,
```

Comment:
I don't think it is easy/intuitive to know what you should pass in options. Is it the name or the id of the account? Is it a path or an ID for the folder?